### PR TITLE
Link to new Cody feedback URL

### PR DESCRIPTION
**Pull request by toolmantim**
## Test plan

- Clicked feedback

### CODE REVIEW

File: vscode/src/chat/protocol.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ export const CODY_DOC_URL = new URL('https://docs.sourcegraph.com/cody')
 // Community and support
 export const DISCORD_URL = new URL('https://discord.gg/s2qDtYGnAE')
 export const CODY_FEEDBACK_URL = new URL(
-    'https://github.com/sourcegraph/cody/discussions/new?category=product-feedback&labels=vscode'
+    'https://github.com/sourcegraph/cody/issues/new/choose'
```
Comment (taylorsperry) [inline vscode/src/chat/protocol.ts:127]: Is this a private repo? Will users be able to access it? @kalanchan

Comment (jdorfman) [reply vscode/src/chat/protocol.ts:127]: @toolmantim that works great. We can convert any issue into a discussion if needed

Comment (toolmantim) [reply vscode/src/chat/protocol.ts:127]: Nope, that’s the same repo @taylorsperry — just linking to issues instead of discussions.

Comment (taylorsperry) [reply vscode/src/chat/protocol.ts]: Nice!

**Pull request merged**